### DISCUSSION TITLE
"from dataset" appended to column names when exporting .fcs

### DISCUSSION THREAD

**Discussion by hrj21** [ImmuneDynamics/Spectre#46] [Help]
Hiya,

I know I've been commenting a lot lately sorry, it's only because I'm using Spectre quite intensely as I love it so much (so please let me know how I can help in return for all my questions and whining).

When using `write.files(write.csv = TRUE)`, all of the parameter names get "_column [column name] from dataset" appended to them. Is there a reason for this? I think I would prefer the column names to be preserved exactly as they appear in the data.table. Perhaps there is a reason for this I'm missing.

Thanks as always for your hard work on this package.
Hefin

**Comment by tomashhurst** [comment]
Hey @hrj21 , thanks for your kind feedback! Please keep up the commenting :).

Do you mean the names that end up in the FCS file? Indeed the '_column [column name] from dataset' is I guess some baggage from back when we were first learning to manipulate flowFrames. Originally it came about to differentiate the text in the _stain_ vs the _parameter_ keywords. I like the idea of removing the extra text, I'll push an update to adjust it.

Tom

**Reply by hrj21** [reply]
Hey Tom! Don't suppose you've got any updates on this?
